### PR TITLE
refactor: Migrate entire project to use Safe Args

### LINKED ISSUE #1205: Migrate entire project to use SafeArgs in fragments instead of raw bundles
**Describe the bug**
Large parts of the app currently use raw bundles to access and pass arguments. This is unsafe and does not utilize the Navigation Architecture Component to its fullest extent.

**Additional context**
Since we already use the Navigation architecture component, it is easier to migrate to using SafeArgs. This will also be benficial if we decide to use Actions in the Nav AAC to switch between different screens.

**Would you like to work on the issue?**
- [x] Yes

### PR DESCRIPTION
**Pull request by haroldadmin**
Fixes #1205 

Changes: Replaced existing fragment args with safe args. Default values
nullability has been inferred by me based on what values were being
passed earlier.

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt [hunk 1]
```diff
@@ -108,12 +108,8 @@ class AttendeeFragment : Fragment() {
...
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        val bundle = this.arguments
-        if (bundle != null) {
-            id = bundle.getLong(EVENT_ID, -1)
-            eventId = EventId(id)
-            ticketIdAndQty = bundle.getSerializable(TICKET_ID_AND_QTY) as List<Pair<Int, Int>>
-        }
+        eventId = EventId(safeArgs.eventId)
+        ticketIdAndQty = safeArgs.ticketIdAndQty as List<Pair<Int, Int>>
```
[the remark this answers is not in the document]
Comment (haroldadmin) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: For reviewers of this PR, here's a look into why I did not change this from the way it was being done before. There's a limitation on the types of data you can pass through safe args too. Primitives work well, but if you need to send complex data types then you do either of the following things:

1. Pass the object as a serializable and then convert it back to its original type on the receiver's end. This is an unsafe cast, though.
2. Create a wrapper class around the object you want to pass and make that wrapper class implement the type Serializable. If required, I can create an additional data class like TicketIdAndQtyWrapper that implements serializable interface, and pass this to the other fragment to unbox its value. **However, this involves changing code that I felt was beyond the scope of this PR**

**Comment by liveHarshit** [comment]
Build is failing.

**Comment by haroldadmin** [comment]
I have fixed the wildcard imports issue, along with the other spotlessKotlin issue. I have also added documentation to the wrapper data class I created.

Cool trick I learned from the build log is to execute `./gradlew spotlessApply` before pushing any commit. Will use this in every future contribution!

**Comment by haroldadmin** [comment]
@iamareebjamal I think I have passed all the PR checks. Please review this PR when you can 😊

**Pull request merged**